### PR TITLE
Fix `torch.max` optional args `dim`, `keepdim` description

### PR DESCRIPTION
**Pull request by zeshengzong**
[`torch.max`](https://pytorch.org/docs/stable/generated/torch.max.html#torch.max) optional args `dim`, `keepdim` not described in document, but users can ignore them.

```python
>>> import torch
>>> a = torch.randn(3,1,3)
>>> a.max()
tensor(1.9145)
>>> a.max(dim=1)
torch.return_types.max(
values=tensor([[ 1.1436, -0.0728,  1.3312],
        [-0.4049,  0.1792, -1.2247],
        [ 0.8767, -0.7888,  1.9145]]),
indices=tensor([[0, 0, 0],
        [0, 0, 0],
        [0, 0, 0]]))

```

## Changes

- Add `optional` description for `dim`, `keepdim`
- Add example of using `dim`, `keepdim`

### Before

image

### After

image

### CODE REVIEW

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -6499,13 +6504,22 @@ def merge_dicts(*dicts):
...
     torch.return_types.max(values=tensor([0.8475, 1.1949, 1.5717, 1.0036]), indices=tensor([3, 0, 0, 1]))
+    >>> a = torch.randn(3,1,3,)
+    >>> a.max(dim=0, keepdim=True)
+    torch.return_types.max(
+    values=tensor([[[1.2928, 1.4774, 0.8201]]]),
+    indices=tensor([[[2, 2, 1]]]))
+    >>> a.max(dim=0, keepdim=False)
+    torch.return_types.max(
+    values=tensor([[1.2928, 1.4774, 0.8201]]),
+    indices=tensor([[2, 2, 1]]))
```
Comment (colesbury) [inline torch/_torch_docs.py:6515]: Please use an example that makes the behavior more clear. For example, construct an input tensor using `torch.tensor()` with regular values. Integer values (e.g., 1.0, 2.0) make it easier to understand the behavior than random number with four decimal places.

Comment (zeshengzong) [reply torch/_torch_docs.py]: Changed, thanks!

**Review by colesbury — APPROVED** [review]
Thanks!
